src/Patches/GlobalShopController_Patch.cs

```diff
@@ -7,51 +7,56 @@
 namespace custom_item_mod.Patches;
 
 [HarmonyPatch(typeof(GlobalShopController))]
-[HarmonyPatch(nameof(GlobalShopController.Awake))]
 public class GlobalShopController_Awake_Patch
 {
-	private static void Prefix(ref GlobalShopController __instance)
+    [HarmonyPatch(nameof(GlobalShopController.Awake))]
```

Comment:
All of your added code uses 4 spaces as indentation, but this project uses tabs. Please change everything to tabs.

Reply:
Done.

A couple of these files were inconsistent, which led VS to default to spaces for those ones.

It's far more obvious in the diff than in the editor when you've replaced most of the code ;)